src/_pytest/debugging.py

```diff
@@ -338,6 +339,10 @@ def _postmortem_traceback(excinfo):
         # A doctest.UnexpectedException is not useful for post_mortem.
         # Use the underlying exception instead:
         return excinfo.value.exc_info[2]
+    elif isinstance(excinfo.value, ConftestImportFailure):
+        # A config.ConftestImportFailure is not useful for post_mortem.
+        # Use the underlying exception instead:
+        return excinfo.value.excinfo[2]
```

Comment:
nice!  that was easy -- is there a way to demo this in a test?

Reply:
Thanks for the fast feedback! Oh yeah, I will try to write a test for this today.

Reply:
Please take a look at `test_debugging.py`, there are tests there which interact with pdb directly. 👍